src/main/java/com/jfrog/ide/common/exporter/csv/CsvExporter.java

```diff
@@ -0,0 +1,100 @@
+package com.jfrog.ide.common.exporter.csv;
+
+import com.google.common.collect.Lists;
+import com.jfrog.ide.common.exporter.Exporter;
+import com.jfrog.ide.common.exporter.exportable.ExportableViolatedLicense;
+import com.jfrog.ide.common.exporter.exportable.ExportableVulnerability;
+import com.opencsv.bean.HeaderColumnNameMappingStrategy;
+import com.opencsv.bean.HeaderColumnNameMappingStrategyBuilder;
+import com.opencsv.bean.StatefulBeanToCsv;
+import com.opencsv.bean.StatefulBeanToCsvBuilder;
+import com.opencsv.exceptions.CsvDataTypeMismatchException;
+import com.opencsv.exceptions.CsvRequiredFieldEmptyException;
+import org.apache.commons.collections4.comparators.FixedOrderComparator;
+import org.jfrog.build.extractor.scan.DependencyTree;
+import org.jfrog.build.extractor.scan.Issue;
+import org.jfrog.build.extractor.scan.License;
+import org.jfrog.build.extractor.scan.Severity;
+
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author yahavi
+ **/
+public class CsvExporter extends Exporter {
+    static final String SEVERITY_COL = "SEVERITY";
+    static final String IMPACTED_DEPENDENCY_COL = "IMPACTED DEPENDENCY";
+    static final String IMPACTED_DEPENDENCY_VERSION_COL = "VERSION";
+    static final String TYPE_COL = "TYPE";
+    static final String FIXED_VERSION_COL = "FIXED VERSIONS";
+    static final String DIRECT_DEPENDENCIES_COL = "DIRECT DEPENDENCIES";
+    static final String CVES_COL = "CVES";
+    static final String CVSS_V2_COL = "CVSS V2";
+    static final String CVSS_V3_COL = "CVSS V3";
+    static final String ISSUE_ID_COL = "ISSUE ID";
+    static final String SUMMARY_COL = "SUMMARY";
+
+    static final String LICENSE_NAME_COL = "LICENSE";
+
+    private static final List<String> COLUMNS_ORDER = Lists.newArrayList(
+            LICENSE_NAME_COL, SEVERITY_COL, IMPACTED_DEPENDENCY_COL, IMPACTED_DEPENDENCY_VERSION_COL, TYPE_COL,
+            FIXED_VERSION_COL, DIRECT_DEPENDENCIES_COL, CVES_COL, CVSS_V2_COL, CVSS_V3_COL,
+            ISSUE_ID_COL, SUMMARY_COL);
+
+    public CsvExporter(DependencyTree root) {
+        super(root);
+    }
+
+    @Override
+    public String generateVulnerabilitiesReport() throws CsvRequiredFieldEmptyException, CsvDataTypeMismatchException {
+        Writer writer = new StringWriter();
+        StatefulBeanToCsv<CsvVulnerabilityRow> csvWriter = createCsvWriter(CsvVulnerabilityRow.class, writer);
+
+        List<CsvVulnerabilityRow> issueRows = collectVulnerabilities().stream()
+                .map(exportableIssue -> (CsvVulnerabilityRow) exportableIssue)
+                .sorted(Comparator.comparing(issueRow -> Severity.valueOf(issueRow.getSeverity()), Comparator.reverseOrder()))
+                .collect(Collectors.toList());
+        for (CsvVulnerabilityRow issue : issueRows) {
+            csvWriter.write(issue);
+        }
+
+        return writer.toString();
```

Comment:
It may be good to close the writer in a finaly, or use try-withj-resources.
The same goes for the ```generateViolatedLicensesReport``` method.